src/penn_chime/models.py

```diff
@@ -44,6 +44,14 @@ def __init__(self, p: Parameters) -> SimSirModel:
             p.known_infected / infected if infected > 1.0e-7 else None
         )
 
+
+        #self.hospitalized_t = p.current_hospitalized
+        #self.n_days_since_first_hospitalized = p.n_days_since_first_hospitalized
+        if p.n_days_since_first_hospitalized is not None:
+            # do not shorten this to `if foo:` form. if you do and n_days_since_first_hospitalized is 0,
+            # program will crash.
+            p.doubling_time = self.argmin_dt(np.linspace(1, 15, 29), p)
```

Comment:
This is a really horrifying thought, but is there a principled reason the doubling time can never be less than 1 day? Super-spreader events would seem to do exactly that.

Reply:
that's a hardcoded constant from Corey's notebook that we can factor into the `settings.cfg` file if we wish. since we're pushing to prod weekly, we only hope that things don't get that drastic too quickly ;)

Reply:
More broadly, if the parameter sweep from `argmin_dt` returns a value at either extreme of its range, it means either
- The best fit value is approximately the given limit
- The fit failed because the true result is outside the range

We need to not let the latter case go through without warning or examination. It may just as well be an indicator of mis-keyed inputs, as of the math honestly saying anything about the circumstances.

Reply:
There's a physical limit on doubling time down around 0.1, since that implies 10 daily doublings, or about 1000x, which exhausts itself pretty damn quickly. I think there's some threshold higher than that where we should warn the user that the computed value is implausibly low

Reply:
i think you're right. i'll add in some sort of warning system.

Reply:
I believe that a different merge allowed doubling time to be a float/fractional.